src/main/scala/io/keen/client/scala/Client.scala

```diff
@@ -361,6 +366,23 @@ trait Writer extends AccessLevel {
    */
   val writeKey: String = settings.writeKey.getOrElse(throw MissingCredential("Write key required for Writer"))
 
+  // constants
+  val MinSendInterval: Integer = 60
+  val MaxSendInterval: Integer = 3600
```

Comment:
Offhand I'm a little rusty on the semantics in the situation of an anonymous class override, which is currently what's recommended as a way to set `writeKey` at runtime. I'd have to check when I have a few minutes to fully switch contexts to run this branch.

These should at least probably have Scaladoc if they're public.